.github/workflows/build.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         key: build-atrifacts-v1-${{ hashFiles('package-lock.json', 'spago.dhall', 'packages.dhall') }}
     - uses: actions/setup-node@v1
       with:
-        node-version: 10.15
+        node-version: 12.16
```

Comment:
Oops, turns out `jsdom` is not supported on Node v10: https://github.com/jsdom/whatwg-url/issues/209#issuecomment-991947013